moto/kinesisvideo/models.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import unicode_literals
+from boto3 import Session
+from moto.core import BaseBackend, BaseModel
+from datetime import datetime
+from .exceptions import (
+    ResourceNotFoundException,
+    ResourceInUseException,
+)
+import random
+import string
+
+
+class Stream(BaseModel):
+    def __init__(
+        self,
+        region_name,
+        device_name,
+        stream_name,
+        media_type,
+        kms_key_id,
+        data_retention_in_hours,
+        tags,
+    ):
+        self.region_name = region_name
+        self.stream_name = stream_name
+        self.device_name = device_name
+        self.media_type = media_type
+        self.kms_key_id = kms_key_id
+        self.data_retention_in_hours = data_retention_in_hours
+        self.tags = tags
+        self.status = "ACTIVE"
+        self.version = self._get_random_string()
+        self.creation_time = datetime.utcnow()
+        stream_arn = "arn:aws:kinesisvideo:{}:123456789012:stream/{}/1598784211076".format(
```

Comment:
I'm assuming this is the account ID? Let's import that from `moto.core`, to avoid magic numbers as much as possible

Reply:
https://github.com/spulec/moto/pull/3271/commits/41a2eb0c6a3a21fe717ae4a6334d476b1515fe20